whynotearth/BrowTricks
# Feature: CSS Loading Spinner

# Description

We include code for a pure CSS loading spinner, mostly ported from [this](https://github.com/whynotearth/Volkswagen-email-corporate-communications-CMS/pull/257) PR from Volkswagen.

The spinner will be used to display on the screen while data is being fetched.

# How to Test/Reproduce

See [here](https://deploy-preview-61--browtricks.netlify.app/loading-spinner-test) for a test route that I have set up to see the spinner in action.

# Screenshots/casts

![image18](https://user-images.githubusercontent.com/20192983/83108462-b31e9d80-a074-11ea-8308-da09cd233eec.gif)

This is a low-FPS non-perfectly looped screencast, to see it in the actual app see the link in the deploy preview above.

# Additional Context

One specific thing I wasn't certain on was the class name for the spinner. I went with `loading` but if anyone has suggestions on a better name then let me know.
